docs/source/getting-started/installing.adoc

```diff
@@ -65,7 +65,8 @@ If you encounter issues related to the hypervisor, see the xref:../using/trouble
 
 [NOTE]
 ====
-- On Windows operating system, due to issue link:https://github.com/minishift/minishift/issues/236[#236], you need to execute the minishift binary from the drive containing your *%USERPROFILE%* directory.
+- On Windows operating system, due to issue link:https://github.com/minishift/minishift/issues/236[#236], you need to execute the Minishift binary from your local C:/ drive.
+You cannot run Minishift from a network drive.
```

Comment:
Exactly. Some folks might turn around and say "well I tried to run it from my local C:/ but connecting to it from another computer via ssh" or something to that affect, so we need to make sure that we're extra-explicit about not running it from a network location.